hooks/tk-multi-loader2/basic/scene_actions.py

```diff
@@ -149,14 +162,18 @@ def execute_action(self, name, params, sg_publish_data):
         if name == "smart_reference":
             self.create_smart_reference(path)
 
-        elif name == "import":
-            vrFileIO.loadGeometry(path)
+        elif name == "import": #TODO: move to function
+            parent = vrScenegraph.getRootNode()
+            vrFileIOService.importFiles([path], parent)
+
+        elif name == "import_with_options": #TODO: move to function
+            vrGUIService.openImportDialog([path])
 
         elif name == "import_sceneplate":
             image_path = self.get_publish_path(sg_publish_data)
             self.import_sceneplate(image_path)
 
-    def execute_multiple_actions(self, actions):
+    def execute_multiple_actions(self, actions): #TODO: make multi-select work
```

Comment:
Just curious, what do you mean by "make multi-select work"?

The function looks like it should execute multiple actions. Do you mean that the loader itself should enable multi-select? If that's the case, then I would suggest moving this comment to the loader app, or maybe specify that it is the loader app that needs to change, and not specifically this function.